include/utility/json_utility.hpp

```diff
@@ -372,7 +372,7 @@ inline bool processGpioPresenceTag(const nlohmann::json& i_parsedConfigJson,
             std::nullopt, std::nullopt, std::nullopt, std::nullopt);
 
         logging::logMessage(l_errMsg);
-        return false;
+        return true;
```

Comment:
Add a detailed comment before this else it can confuse in future as why we are returning true in case of exception.

Reply:
Done